text/0018-genesis_block.md

```diff
@@ -0,0 +1,94 @@
+- Proposal Code Name: genesis_block
+- Start Date: 2019-09-27
+- NEP PR: [nearprotocol/neps#0000](https://github.com/nearprotocol/neps/pull/0000)
+
+# Summary
+[summary]: #summary
+
+Genesis block will contain:
+* Accounts and smart contracts required to describe the initial ownership, vesting schedule, and the lockup of the tokens;
+* Economics config;
+* Accounts from the testnet.
+
+# Token ownership
+We want Near investors and token holders to have their vesting schedule, lock up period and ability to stake to be implemented through the accounts and smart contracts on the blockchain.
+
+Each token holder will have tokens that go through the following lifecycle:
+* Unvested;
+* Vested, but locked;
+* Unlocked.
+
+Vesting is done over 4 years with 1 year cliff starting with:
```

Comment:
I think investors vest over 2 years with no cliff.

Reply:
Also people in beta program and ambassador program have a different vesting schedule

Reply:
Fixed.